tests/test_feature.rb

```diff
@@ -51,7 +51,7 @@ def feature(feat)
 
     # Get current state of feature, then disable it
     pre_clean_enabled = Feature.send("#{feat}_enabled?")
-    config("no #{feat_str}") if pre_clean_enabled
+    config("no feature #{feat_str}") if pre_clean_enabled
```

Comment:
The fix for this was committed yesterday under another PR so you can just update your workspace to get rid of these diffs.